docs/devsite-help/troubleshooting.md

```diff
@@ -141,6 +141,35 @@ In that case, the simplest fix is to add a direct dependency to
 `Grpc.Core` from your application, which will ensure that the
 native libraries are copied appropriately.
 
+## How to get ErrorInfo from SpannerException
```

Comment:
No, "How to get..." isn't a question - it's just information. (It would be "How can I get ErrorInfo from SpannerException?" or similar.)

But yes, I'd make this primarily about RpcException.

(This is my one drive-by comment...)